pyroengine/inference/inference.cfg

```diff
@@ -0,0 +1,13 @@
+[MODEL]
+backbone = rexnet1_0x
+num_classes = 1
+use_adaptive_concat_pool2d_head = True
+nb_features = 1280
+cut = -2
+checkpoint = pyro_checkpoint_V0.1.pth
```

Comment:
Shouldn't the checkpoint either be passed by the user (depending on local path) or be a remote publicly accessible URL?